week5/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM python:latest as base
+LABEL maintainer="lyabomyr@gmail.com"
+RUN pip install --upgrade pip
+
+WORKDIR /ml_project
+COPY week5/requirements.txt /ml_project/requirements.txt
+RUN pip install --no-cache-dir --upgrade -r  /ml_project/requirements.txt
+
+
+ENV WANDB_PROJECT="liunomyr"
+ENV WANDB_API_KEY="0a14fd83112d8258ad9cb780edb9e9932241f5fc"
+
+ENV PYTHONPATH="$PWD/ml_project"
+COPY week5/serving /ml_project/serving
+COPY week5/unit_tests /ml_project/unit_tests
+COPY Churn_Modelling.csv /ml_project/Churn_Modelling.csv
+COPY week5/test_churn_modelling.csv /ml_project/test_churn_modelling.csv
+
+RUN echo $PYTHONPATH
+RUN pytest unit_tests/test_fastapi.py
```

Comment:
let's not call this here, and call in CI as separate step, independent from building process

Reply:
ok, fixed